core/management/commands/new_event.py

```diff
@@ -6,13 +6,11 @@
 from django.template.loader import render_to_string
 from slacker import Error as SlackerError
 
-from core.command_helpers import gather_event_date_from_prompt
-from core.default_eventpage_content import (get_default_eventpage_data,
-                                            get_default_menu)
-from core.forms import AddOrganizerForm
-from core.models import Event, EventPageContent, EventPageMenu
-from core.slack_client import slack
-from core.utils import get_coordinates_for_city
+from ...command_helpers import gather_event_date_from_prompt
+from ...forms import AddOrganizerForm, EventForm
+from ...models import Event
+from ...slack_client import slack
+from ...utils import get_coordinates_for_city
```

Comment:
This is something I proposed. It's good because it makes the imports independent of the package name. If we decide to rename the `core` app it will help.

As for renaming the name of this app should be something more meaningful, like `events` and the User related models and views should be moved to some other app. Currently when people don't know where to add something (like view or a model) they add it to core and we end up with this huge app with basically everything. Moving models around between apps can be done with migrations and it's pretty straightforward, so I would be happy to do it in some point in the future :)